.github/workflows/publish.yaml

```diff
@@ -119,6 +125,10 @@ jobs:
             -o $(find .make_cache -name "*.tag")
       
           IMAGE_NAME=$(make help FLAVOUR=jupyter REPOSITORY=${REPOSITORY} TAG=${TAG} help | grep "Image\:" | cut -d ":" -f2  | xargs)
+
+          DESCRIPTION=$(sed -n '/^#\ \-/,/^#\ \-/  p' build/Dockerfile.jupyter | sed 's/^\#\ //g' | sed '$ d' | tail -n+2 | xargs)
```

Comment:
suggestion: Can we add a comment here describing what we're trying to do? This is very hard to read and understand.